website/content/en/docs/getting-started/getting-started-with-karpenter/_index.md

```diff
@@ -79,6 +79,10 @@ The following cluster configuration will:
 
 {{% script file="./content/en/{VERSION}/getting-started/getting-started-with-karpenter/scripts/step06-add-spot-role.sh" language="bash"%}}
 
+{{% alert title="Note" color="primary" %}}
```

Comment:
Change this to something like-

> In order to run Windows workloads, Windows support should be enabled in your EKS Cluster. See documentation here for enabling Windows support.

The instructions regarding what to do for enabling support should come from a single destination. We should not document the specific things to do since they are just a part of a wider set of steps.